labellab-flask/api/controllers/labelscontroller.py

```diff
@@ -0,0 +1,218 @@
+from flask.views import MethodView
+from flask import make_response, request, jsonify, current_app
+from flask_jwt_extended import (
+    jwt_required,
+    get_jwt_identity,
+    get_raw_jwt
+)
+from api.models.Label import Label
+from api.helpers.label import (
+    find_by_id, 
+    find_by_label_name, 
+    find_all_by_project_id,
+    save as save_label, 
+    delete_by_id as delete_label,
+    update_label
+)
+
+
+class CreateLabel(MethodView):
+    """This class creates a new Label."""
+    @jwt_required
+    def post(self, project_id):
+        """
+        Handle POST request for this view.
+        Url --> /api/v1/label/create/<int:project_id>
+        """
+        # getting JSON data from request
+        post_data = request.get_json(silent=True,
+                                     force=True)
+        try:
+            label_name = post_data["label_name"]
+            label_type = post_data["label_type"]
```

Comment:
Should validate the content of ```label_type```. In my opinion, let's keep ```rectangle``` and ```polygon``` as supported types and update when we implement other types of labels. @carpecodeum what's your idea?